src/Request.php

```diff
@@ -153,12 +152,18 @@ protected function throwServerError($body, $httpStatusCode)
             $httpStatusCode);
     }
 
-    protected function getRequestHeaders()
+    protected function getRequestHeaders($additionHeader)
     {
+        if(isset($additionHeader)){
```

Comment:
dont check isset. it will always return true since the variable is set. check if(empty(additionHeader) === false please